src/commonMain/resources/shader-libraries/built-in/movers/Movers Beats.glsl

```diff
@@ -0,0 +1,144 @@
+// On each beat jumps an eye to a random position. Can be constrained to a horizon, sky or ground.
+// by Aravindo Wingeier
+
+struct FixtureInfo {
+    vec3 position;
+    vec3 rotation;
+    mat4 transformation;
+};
+
+struct MovingHeadParams {
+    float pan;
+    float tilt;
+    float colorWheel;
+    float dimmer;
+};
+
+uniform FixtureInfo fixtureInfo;
+
+struct BeatInfo {
+    float beat;
+    float bpm;
+    float intensity;
+    float confidence;
+};
+uniform BeatInfo beatInfo; // @@baaahs.BeatLink:BeatInfo
+
+uniform float time; // @@Time
+uniform bool sky; // @@Switch enabled=true
```

Comment:
Love this! I'm gonna see if I can pull in the `Select` widget from [this branch](https://github.com/baaahs/sparklemotion/pull/508), which gives radio button-like behavior.